### ISSUE TITLE
Click "Passthrough" stops working when disabling title bar

### ISSUE THREAD

**Issue by kylebelt** [stek29/Helium#7]
Mac OS El Capitan, using Release 2.0.2

Whenever I disable title bar the the clicking Helium windows will always bring it back on focus instead of passing through the app under it.

I have to restart the application in order to have it working again (it will restart with title bar enabled).

PS: Thanks for the great job of keeping this project alive.

**Comment by stek29** [comment]
I've reproduced this, and I'm not sure what's causing it.

I'm thinking about getting rid of title bar at all, probably adding dragging pad in corner, since it causes too much problems.

Unfortunately, I have exam session now, so I don't have enough time really. But it should end in a week or two ;)

**Comment by kylebelt** [comment]
> I'm thinking about getting rid of title bar at all, probably adding dragging pad in corner, since it causes too much problems.

I agree with this. I think the title bar is wasted space and I only use it for resizing (which the dragging pad would address anyway)
